test/variational/test_continuous_expect.py

```diff
@@ -40,19 +40,19 @@ def v2(x):
 
 model = test()
 model2 = test2()
-vs_continuous = nk.vqs.MCState(sab, model, n_samples=10 ** 6, n_discard_per_chain=2000)
+vs_continuous = nk.vqs.MCState(sab, model, n_samples=10**6, n_discard_per_chain=2000)
 vs_continuous2 = nk.vqs.MCState(
-    sab, model2, n_samples=10 ** 7, n_discard_per_chain=2000
+    sab, model2, n_samples=10**7, n_discard_per_chain=2000
 )
 
 
 def test_expect():
-    x = vs_continuous2.samples.reshape(-1, 1)
+    # x = vs_continuous2.samples.reshape(-1, 1)
     sol = vs_continuous.expect(pot)
     O_stat, O_grad = vs_continuous2.expect_and_grad(e)
     O_grad, _ = nk.jax.tree_ravel(O_grad)
 
-    O_grad_exact = 2 * jnp.dot(x.T, (v1(x) - jnp.mean(v1(x), axis=0))) / x.shape[0]
+    # O_grad_exact = 2 * jnp.dot(x.T, (v1(x) - jnp.mean(v1(x), axis=0))) / x.shape[0]
```

Comment:
What was the issue with this test?

Reply:
`x` and `O_grad_exact ` are not used, so `flakehell` will report errors

Reply:
wait, are we really sure it is safe to remove this test? @gpescia

Reply:
Yes I think Fillipo added this when we had the issue with the factor of 2 in the gradients for real wave-functions, and never removed it. Since we test against the exact gradient of the energy (-1.4...) we don't need it I'd say.

Reply:
Nopey nope. I just removed the beautiful factor of two you had a few lines below. 
https://github.com/netket/netket/blame/75a602af1be430cedf2faee95c1745f08db7ce40/test/variational/test_continuous_expect.py

This line here never did anything ever.

Could you maybe do a tiny PR where you test the gradient, instead of against a magic number (-0.140) against the `O_grad_exat` you are computing here? I guess it should evaluate to -0.140..

Reply:
Ah I see, my bad..
Hmm I think testing agains the true exact value (-0.140) is stronger than (and includes) testing against the exact estimator of the true value, no?